cmd/substreams/pack.go

```diff
@@ -20,12 +23,45 @@ var packCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(packCmd)
+	packCmd.Flags().StringP("output-dir", "o", ".", cli.FlagDescription(`
+		Optional flag to specify output directory to output generated .spkg file. 
+		If the received argument starts with a "." it will be treated as a local path relative to the supplied manifest path.
+	`))
+}
+
+func is_relative_path(path string) bool {
+	return path[0:1] == "."
 }
 
 func runPack(cmd *cobra.Command, args []string) error {
+	outputDir := maybeGetString(cmd, "output-dir")
+	if outputDir == "." {
+		outputDir = ""
+	}
```

Comment:
Not needed anymore, reality is that "." and "" are equivalent when making relative path absolute base on working directory see other bigger comment.
```suggestion
```

Reply:
Have now removed